### PR TITLE
Feat/watch later

### PR DESCRIPTION
**Pull request by purvasharmadev**
## Added Watch later feature

- add a video to watch later
- remove a video from watch later

### CODE REVIEW

**Review by adityapandadev — APPROVED** [review]

File: src/Pages/Nav/nav.jsx [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 function Nav() {
+  const { isLoggedIn, logOut } = useAuth();
 
-  const {isLoggedIn,logOut} = useAuth()
-
-    // function to toggle on small screen
+  // function to toggle on small screen
   function ClickHandler() {
     const navList = document.getElementsByClassName("navbar-list")[0];
     return navList.classList.toggle("toggle-active");
```
Comment (tonymike99) [inline src/Pages/Nav/nav.jsx:14]: Suggestion: We should not use getElementsByClassName in React since it is against the declarative concept of React..

**Review by tonymike99 — APPROVED** [review]
HI Purva,
The code is fine. Please look into the suggestion and make the changes accordingly.

**Review by varun13169 — APPROVED** [review]
Code is neat and file arrangement is done pretty well.

**Comment by purvasharmadev** [comment]
> HI Purva, The code is fine. Please look into the suggestion and make the changes accordingly.

Thank you for you suggestions, will work on it.

**Pull request merged**